azure_functions_worker/bindings/shared_memory_data_transfer/shared_memory_manager.py

```diff
@@ -170,7 +171,7 @@ def free_mem_map(self, mem_map_name: str) -> bool:
                 f'Cannot find memory map in list of allocations {mem_map_name}')
             return False
         shared_mem_map = self.allocated_mem_maps[mem_map_name]
-        success = shared_mem_map.dispose()
+        success = shared_mem_map.dispose(is_delete_backing_resources)
```

Comment:
This flag if it is disposed of or not. Correct?

Reply:
Can you have a better naming for this? The "is_delete" part was confisuing. It is more like "to_delete..."?